### PR TITLE
use typeorm migrations for quickstart

### PR DESCRIPTION
**Pull request by depombo**

### CODE REVIEW

**Review by dfellis — APPROVED** [review]

File: docs/quickstart.md [hunk 1]
```diff
@@ -109,28 +109,62 @@ $ iasql install --db startup aws_cloudwatch aws_ecr aws_ecs aws_elb aws_security
...
 ```
 
-3. Run the script on your db by invoking `psql` with the connection string provided on db creation and set the desired `project-name` parameter that your resources will be named after:
+2. Install the `typeorm` CLI and the Node.js project that uses TypeORM under the `quickstart/infra` folder
 
 ```bash
-psql postgres://d0va6ywg:nfdDh#EP4CyzveFr@db.iasql.com/_4b2bb09a59a411e4 -v project_name="'quickstart'" -f quickstart.sql
+cd infra
+npm i
+npm -g i typeorm
```
Comment (aguillenv) [inline docs/quickstart.md:124]: Why is this needed? would not the `npm i` install TypeORM?

File: docs/quickstart.md [hunk 2]
```diff
@@ -109,28 +109,62 @@ $ iasql install --db startup aws_cloudwatch aws_ecr aws_ecs aws_elb aws_security
...
+3. (Optional) Set the desired project name that your resources will be named after by changing the `name` in the `quickstart/infra/package.json`. If the name is not changed, `quickstart` will be used.
+
 :::note
 
 The `project-name` can only contain alphanumeric characters and hyphens(-) because it will be used to name the load balancer
 
 :::
 
-4. Apply the changes described in the db to your cloud account
+3. Create a [`ormconfig.json`](https://typeorm.io/#/using-ormconfig/using-ormconfigjson) with the connection parameters provided on db creation. In this case:
```
Comment (aguillenv) [inline docs/quickstart.md:135]: maybe we can provide this file with all fields commented?

File: docs/quickstart.md [hunk 3]
```diff
@@ -109,28 +109,62 @@ $ iasql install --db startup aws_cloudwatch aws_ecr aws_ecs aws_elb aws_security
...
+   },
+   "logging": false,
+   "migrations": [
+      "src/migration/**/*.js"
+   ],
+   "cli": {
+      "migrationsDir": "src/migration"
+   }
+}
+}
```
Comment (aguillenv) [inline docs/quickstart.md:158]:
```suggestion
```

File: docs/quickstart.md [hunk 4]
```diff
@@ -109,28 +109,62 @@ $ iasql install --db startup aws_cloudwatch aws_ecr aws_ecs aws_elb aws_security
...
+```
+
+4. Run the existing TypeORM migration on your IaSQL db by invoking `typeorm` CLI
+
+```bash
+typeorm migration:run
```
Comment (aguillenv) [inline docs/quickstart.md:164]: Oh well if you follow my suggestion of inly use `npm I` this would be:
```suggestion
npx typeorm migration:run
```

Comment (dfellis) [reply docs/quickstart.md:164]: 💯

File: docs/quickstart.md (hunk 2 above)
Comment (depombo) [reply docs/quickstart.md:135]: we display it below and Docusaurus gives you the copy prompt :)
